### DISCUSSION TITLE
Personal Collection area - exploration

### DISCUSSION THREAD

**Discussion by NickyPhelanBBC** [guardian/grid#4072] [Ideas]
Here are the slides I presented at Grid Hour - outlining why we're exploring a personal Collections area.

Once we've got some more thinking to show I'll let you know (hopefully at the next Grid Hour).

Feel free to add thoughts and comments!

[BBCImages-Collections-GuardianDeckApril.pdf](https://github.com/guardian/grid/files/11310131/BBCImages-Collections-GuardianDeckApril.pdf)

**Comment by paperboyo** [comment]
Just in case this is in scope (if not for fixes, then just for discovery), prompted by one of our users reporting a long-standing issues with Collections just now, I will keep all known issues with them in this comment. FYI, if anything else.

- [ ] no way of deleting a collection with subcollection(s)
- [ ] no way of renaming the collection (other than creating one with new name, adding from old, deleting from old, deleting old)
- [ ] `id` is the name. Two collections with the same name that differs only by capitalisation, break the system
- [ ] When one deletes the collection with images in it, the information about them being in this (now deleted from sidebar) collection stays in images metadata. Good for undo. Ungood for data cleanliness.
- [ ] collection batch operations haven’t gone through Angular speedups ([this](https://github.com/guardian/grid/pull/3064) and adjacent) that other operations went through, so they are terribly slow
- [ ] collection operations are not available via the Info panel when multiple images are selected (only if one is), so the only way to do them is via drag&drop. As all other operations, batch editing should be available in the Info panel if only for discoverability

**Reply by NickyPhelanBBC** [reply]
- we're proposing that only the owner of a personal collection will be able to delete it
- if we allow collection owners to rename their personal collections then we should consider enabling the same functionality for main collections
- sounds like we would need to do some work around the id of the collections as I imagine multiple people may create folders with the same name, eg fave pics or something

**Reply by NickyPhelanBBC** [reply]
Here are the wireframes from today. [Grouping-images-wireframes-guardian.pdf](https://github.com/guardian/grid/files/11530513/Grouping-images-wireframes-guardian.pdf)

Thanks very much for your feedback, we'll keep developing the wireframes and feeding back to you. We're also planning to do some light-touch user testing over the next few weeks with some of our user group, to get some early feedback, we'll share this too.

Any further thoughts please share! Thanks.
